OpenSDS_Volume_Actions_Design.md

```diff
@@ -0,0 +1,92 @@
+# Title
+
+Status: Pending
+
+Version: Beta
+
+Author(s): PengYiOfOpenSDS
+
+## Summary
+
+Extends the size of a volume. 
+
+## Motivation
+
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
+### Goals
+
+What will be addressed by the proposal?
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
+### Non-Goals
+
+What will not be addressed by the proposal?
+"Shrinks the size of a volume" will not be addressed.
+
+## Proposal
+
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
+### Data model impact
+
+Does the proposal affect data model?
+No
+
+### REST API impact
+
+Does the proposal affect API?
+v1/{ProjectID}/ volumes/{VolumeID} /action
+Method: POST
+JSON schema definition:
+{
+"extend": {
+"newSize": 3
+}
+}
+Successful response code: 202
```

Comment:
Please fix the format of this section so that it shows up right.  See how it looks like now below.

https://github.com/PengYiOfOpenSDS/design-specs/blob/2dcb6ceddb20448604e392ab999a43c666adfa5a/OpenSDS_Volume_Actions_Design.md